src/framework/uicomponents/qml/MuseScore/UiComponents/internal/StyledMenu.qml

```diff
@@ -138,6 +139,8 @@ MenuView {
                 case NavigationEvent.Right:
                     var selectedItem = prv.selectedItem()
                     if (!Boolean(selectedItem) || !selectedItem.hasSubMenu) {
+                        root.close(true)
```

Comment:
We also use StyledMenu for the context menu in notation view. 
For the context menu we shouldn't close the menu by pressing Right/Left. 
I think we should provide property for controlling such behavior - like hasSiblings

Reply:
What would Sibling mean in this context? If two menus are open at the same time?
And where is the context menu for notation view?